sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/data/repository/RepositoryImp.kt

```diff
@@ -50,6 +52,8 @@ class RepositoryImpl @Inject constructor() : Repository {
         const val MOLECULE_BOLD_TITLE_BODY_VIEW = 5
         const val MOLECULE_INSET_VIEW = 6
         const val MOLECULE_INSET_TEXT_VIEW = 7
-        const val MOLECULE_MULTI_COLUMN_ROW_VIEW = 8
+        const val MOLECULE_TEXT_INPUT_VIEW = 8
+        const val MOLECULE_CURRENCY_INPUT_VIEW = 9
```

Comment:
Would be nice to keep these in the same order as they are on the screen, at least for this initial phase!
```suggestion
        const val MOLECULE_TEXT_INPUT_VIEW = 1
        const val MOLECULE_CURRENCY_INPUT_VIEW = 2
```